docs/HowTo/Configure-TLS.md

```diff
@@ -0,0 +1,118 @@
+[`tls-known-clients-file`](../Reference/CLI/CLI-Syntax.md#tls-known-clients-file) option.
+* Allow access to clients with trusted CA certificates using the [`--tls-allow-ca-clients`](../Reference/CLI/CLI-Syntax.md#tls-allow-ca-clients) option.
+
+!!! note
+    Use the [`--tls-allow-any-client`](../Reference/CLI/CLI-Syntax.md#tls-allow-any-client) option to allow
+    access to any client that presents a certificate. 
+    
+    [`--tls-allow-any-client`](../Reference/CLI/CLI-Syntax.md#tls-allow-any-client) cannot be used with [`tls-known-clients-file`](../Reference/CLI/CLI-Syntax.md#tls-known-clients-file) or [`--tls-allow-ca-clients`](../Reference/CLI/CLI-Syntax.md#tls-allow-ca-clients).
+    
+## Server TLS Connection
+
+Allow EthSigner to to send and receive secure HTTP JSON-RPCs from the
+server (for example Besu).
+
+**Prerequisites**:
+
+* The server must be configured to allow TLS communication
+* Server's password-protected PKSC #12 keystore information.
+
+
+### Create the Known Servers file
+
+Create a file (in this example, `knownServers`) that lists one or more trusted
+servers. The file contents use the format `<hostame>:<port> <hex-string>`
+where:
+
+* `<hostname>` is the server hostname
+* `<port>` is the port used for communication
+* `<hex-string>` is the SHA-256 fingerprint of the server's certificate.
+
+!!! example
+    ```
+    localhost:8590 6C:B2:3E:F9:88:43:5E:62:69:9F:A9:9D:41:14:03:BA:83:24:AC:04:CE:BD:92:49:1B:8D:B2:A4:86:39:4C:BB
+    127.0.0.1:8590 6C:B2:3E:F9:88:43:5E:62:69:9F:A9:9D:41:14:03:BA:83:24:AC:04:CE:BD:92:49:1B:8D:B2:A4:86:39:4C:BB
+    ```
+
+!!! note
+    Specify both hostname and IP address in the file if unsure which is used in
+    requests.
+    
+### Start EthSigner
+
+```bash
+ethsigner --downstream-http-tls-enabled --downstream-http-tls-keystore-file=/Users/me/my_node/keystore.pfx --downstream-http-tls-keystore-password-file=/Users/me/my_node/keyPassword --downstream-http-tls-known-servers-file=/Users/me/my_node/knownServers
+```
+
+The command line:
+
+* Enables TLS using the [`--downstream-http-tls-enabled`](../Reference/CLI/CLI-Syntax.md#downstream-http-tls-enabled)
+option.
+* Specifies the keystore to present during authentication using the 
+[`--downstream-http-tls-keystore-file`](../Reference/CLI/CLI-Syntax.md#downstream-http-tls-keystore-file) option.
+* Specifies the file that contains the password to decrypt the keystore using
+the [`--downstream-http-tls-keystore-password-file`](../Reference/CLI/CLI-Syntax.md#downstream-http-tls-keystore-password-file) option.
+* [Specifies the servers](#create-the-known-servers-file) to connect to using the [`--downstream-http-tls-known-servers-file`](../Reference/CLI/CLI-Syntax.md#downstream-http-tls-known-servers-file) option.
+
+!!! note
+    The [`downstream-http-tls-ca-auth-enabled`](../Reference/CLI/CLI-Syntax.md#downstream-http-tls-ca-auth-enabled) option is enabled
```

Comment:
does `downstream-http-tls-ca-auth-enabled` needs to be preceded with `--`?

Reply:
Done